targets/nvidia-jetson-orin/optee.nix

```diff
@@ -82,15 +82,35 @@
           if config.ghaf.hardware.nvidia.orin.optee.xtest
           then
             (
-              let
-                xTestTaDir = "${opteeXtest}/ta";
-              in
-                builtins.map (
+              builtins.map (
                   ta: {
                     name = "optee_armtz" + "/" + ta;
-                    path = xTestTaDir + "/" + ta;
+                    path = "${opteeXtest}/ta" + "/" + ta;
                   }
-                ) (builtins.attrNames (builtins.readDir xTestTaDir))
+                ) [
+                  # List of OP-TEE's xtest required TA's
+                  # List gathered from OP-TEE's test sources
+
```

Comment:
Could you please add instructions or script, how to obtain and update this list?

Reply:
Nice! Thanks :+1: